department-of-veterans-affairs/abd-vro
# MCP-2288 pdfTemplate Default value

## What was the problem?

Not an issue exactly but the PDF Generator had extra logic to set `pdfTemplate` if it wasn't passed in the request. Since the generator should only do PDF related work, this logic was moved to the Java side

Associated tickets or Slack threads:

- [MCP-2288](https://amida.atlassian.net/browse/MCP-2288)

## How does this fix it?[^1]

Code move to Java

## How to test this PR
- Using Swagger, generate a PDF without passing the `pdfTemplate` variable. It should give you the old V1 template
- As a second test, pass `"pdfTemplate": "v2"` and you should get the V2 template rather than V1


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
